pkg/ingester/shipper.go

```diff
@@ -100,38 +102,28 @@ func NewShipper(
 // to the object bucket once.
 //
 // It is not concurrency-safe, however it is compactor-safe (running concurrently with compactor is ok).
-func (s *Shipper) Sync(ctx context.Context) (uploaded int, err error) {
-	meta, err := readShipperMetaFile(s.dir)
+func (s *Shipper) Sync(ctx context.Context) (shipped int, err error) {
+	shippedBlocks, err := readShippedBlocks(s.dir)
 	if err != nil {
 		// If we encounter any error, proceed with an empty meta file and overwrite it later.
 		// The meta file is only used to avoid unnecessary bucket.Exists call,
 		// which are properly handled by the system if their occur anyway.
-		if !os.IsNotExist(err) {
```

Comment:
Why this was removed?

Reply:
I think because can't happen because this specific error is swallowed by `readThanosShippedBlocks()`.